doc/source/user_guide/copy_on_write.rst

```diff
@@ -29,21 +25,25 @@ pandas indexing behavior is tricky to understand. Some operations return views w
 other return copies. Depending on the result of the operation, mutating one object
 might accidentally mutate another:
 
-.. ipython:: python
+.. code-block:: ipython
+
+    In [1]: df = pd.DataFrame({"foo": [1, 2, 3], "bar": [4, 5, 6]})
+    In [2]: subset = df["foo"]
+    In [3]: subset.iloc[0] = 100
+    In [4]: df
+    Out[4]:
+       foo  bar
+    0  100    4
+    1    2    5
+    2    3    6
 
-    df = pd.DataFrame({"foo": [1, 2, 3], "bar": [4, 5, 6]})
-    subset = df["foo"]
-    subset.iloc[0] = 100
-    df
 
 Mutating ``subset``, e.g. updating its values, also updates ``df``. The exact behavior is
 hard to predict. Copy-on-Write solves accidentally modifying more than one object,
 it explicitly disallows this. With CoW enabled, ``df`` is unchanged:
```

Comment:
Do we want to update the `With CoW enabled` wording?

Reply:
Yep, thx